dev-docs/source/auth.rst

```diff
@@ -95,18 +97,32 @@ gain permission to read the field ``bsn`` on the table ``ingeschrevenpersonen``,
 provided that the request queries for either ``bsn`` and ``lastname``,
 or ``postcode`` and ``lastname`` (or all three fields).
 
-The intention behind ``mandatoryFilterSets`` is to ensure
-that listings are restricted on a need-to-know basis.
-For example, a profile might express that a frontend office employee
-may only access data of someone when they can provide their last name and postal code.
-Or a statistician might be allowed to read age and neighbourhood
-fields to aggregate data, without ever having access to identifiable data.
-Encoding such rules in the schema file using a custom scope
-would require every other request that accesses the dataset to also have this scope.
+The ``mandatoryFilterSets`` ensures that listings are restricted on a need-to-know basis.
+Only when some information can be provided, the API grants access to see the remaining data.
+For example, a frontend office employee may only see data of someone when they can already
+provide their last name and postal code. Or a statistician might be allowed to read age and
```

Comment:
The statistician example isn't context dependent; right? It could saythat the statistician only gets access to certain neigbourhoods e.g. but we don't have that functionality yet so that would be confusing. 

I think what we want to say here though is that the profiles can also be used to grant access to rows and columns similarly to the 'auth' field but without cluttering the main schema needlessly... 

Or do we want to use that example to explain the randomization/first letters only feature?

Reply:
Good point, I've updated this part.